src/notation/internal/notationinteraction.cpp

```diff
@@ -2016,6 +2016,7 @@ void NotationInteraction::endEditText()
     m_textEditData.clearData();
 
     notifyAboutTextEditingChanged();
+    notifyAboutSelectionChanged();
```

Comment:
necessary to ensure that the properties panel (inspector) and menus etc. update correctly when you finish editing a text element using the Esc key (it was working in other cases, such as clicking away, as that action already triggered this notification)